cmd/system-probe/modules/language_detection.go

```diff
@@ -88,7 +80,12 @@ func detectLanguage(writer http.ResponseWriter, request *http.Request) {
 		return
 	}
 
-	resp := toDetectLanguageResponse(languagedetection.DetectWithPrivileges(getRequestPids(&req)))
+	procs := make([]languagemodels.Process, 0, len(req.Processes))
+	for _, proc := range req.Processes {
+		procs = append(procs, proc)
+	}
```

Comment:
suggestion: you will need to import "golang.org/x/exp/slices" for this.
```suggestion
	procs := slices.Clone(req.Processes)
```

Reply:
The reason for the slice copy is to change the type from the [proto impl type](https://github.com/DataDog/datadog-agent/blob/aea395a14e706609ea175ca85d0aa4bf710ed058/pkg/proto/datadog/languagedetection/api.proto#L7-L11) to the [interface `languagemodels.Process`](https://github.com/DataDog/datadog-agent/blob/aea395a14e706609ea175ca85d0aa4bf710ed058/pkg/languagedetection/languagemodels/detector.go#L14), I'm not sure if we can achieve that with `slices.Clone()`?

Reply:
Yeah, I don't think you can.